next.config.js

```diff
@@ -285,7 +285,7 @@ async function redirects() {
     },
     {
       source: '/core/smart/rpc-api.html',
-      destination: '/references/stacks-rpc-api',
+      destination: '/references/stacks-blockchain',
```

Comment:
shouldn't this be `stacks-blockchain-api`?

Reply:
yeah, good call. i didn't review the name but just copy/pasted it. let me rename and add a new redirect